File: StadiumEngine/StadiumEngine.BookingForm/ClientApp/src/models/dto/offers/LockerRoomDto.ts
import {LockerRoomGender} from "./enums/LockerRoomGender";
import {BaseEntityDto} from "../BaseEntityDto";
import {LockerRoomStatus} from "./enums/LockerRoomStatus";

export interface LockerRoomDto extends BaseEntityDto {
    id: number,
    name: string,
    description: string,
    gender: LockerRoomGender,
    isActive: boolean,
    status: LockerRoomStatus
}

File: Unfield/Unfield.DTO/Offers/LockerRooms/LockerRoomDto.cs
using Unfield.Common.Enums.Offers;

namespace Unfield.DTO.Offers.LockerRooms;

/// <summary>
///     ДТО раздевалки
/// </summary>
public class LockerRoomDto : BaseEntityDto
{
    /// <summary>
    ///     Id
    /// </summary>
    public int Id { get; set; }

    /// <summary>
    ///     Название
    /// </summary>
    public string Name { get; set; } = null!;

    /// <summary>
    ///     Описание
    /// </summary>
    public string? Description { get; set; }

    /// <summary>
    ///     Пол
    /// </summary>
    public LockerRoomGender Gender { get; set; }

    /// <summary>
    ///     Активность
    /// </summary>
    public bool IsActive { get; set; }
    
    /// <summary>
    /// Статус
    /// </summary>
    public LockerRoomStatus Status { get; set; }
}